cpp/test/CMakeLists.txt

```diff
@@ -319,6 +319,14 @@ if(BUILD_TESTS)
     test/neighbors/ann_cagra/test_int8_t_uint32_t.cu
     test/neighbors/ann_cagra/test_uint8_t_uint32_t.cu
     test/neighbors/ann_cagra/test_float_int64_t.cu
+    src/neighbors/detail/cagra/search_multi_cta_float_uint64_dim128_t8.cu
```

Comment:
Note: to libraft.so we add precompiled cagra kernels for 32-bit index type only. We have test for 64-bit as well, therefore we add extra object files for the tests here.